alot/widgets/colour_text.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import re
+from ..settings.const import settings
+
+def parse_text_colour(line):
+    """Return colour attribute as (attr, attr_focus)
+
+    :line: line of text to be parsed
+    :returns: attr
+
+    """
+    return parse_quotes(line)
+
+def parse_quotes(line, max_quote_level=7):
+    """Search for quotes. 
+
+    :returns: attr, None if no quote are available
+
+    """
+    default_colour = settings.get_theming_attribute('thread', 'body')
+    is_quoted = False
+    quote_colour = None
+    symbol = settings.get('quote_symbol')
+    if symbol is None:
```

Comment:
use the fallback option when getting the config setting? https://github.com/pazz/alot/blob/master/alot/settings/manager.py#L239
(note that the defaul is "> ", with a trailing space)

Reply:
Does it  make sense here to specify a fallback? If I understand correctly we would still get the default settings value of the variable we're requesting (in this case from `quote_symbol` : `>`), even if no fallback is defined.